xmovie/test/test_core.py

```diff
@@ -10,15 +20,20 @@
     Movie,
 )
 from xmovie.presets import basic, rotating_globe
-import pytest
-import matplotlib as mpl
-import matplotlib.pyplot as plt
-from PIL import Image
-import numpy as np
-import xarray as xr
-import os
-import cv2
-import dask.array as dsa
+
+try:
```

Comment:
These we can most definitely replace similar as in xarray for a cleaner code?

Reply:
done